pkg/crc/machine/machine.go

```diff
@@ -372,12 +372,12 @@ func Start(startConfig StartConfig) (StartResult, error) {
 		}
 	}
 
-	// Start kubelet inside the VM
+	// Check if kubelet service is running inside the VM
 	sd := systemd.NewInstanceSystemdCommander(sshRunner)
-	kubeletStarted, err := sd.Start("kubelet")
+	kubeletStarted, err := sd.IsActive("kubelet")
```

Comment:
This only leaves `configureCluster` to start the kubelet service inside the VM. This function is only called when creating a VM, not when starting a pre-existing VM. This means `crc start; crc stop; crc start` now fails
```
INFO Check DNS query from host ...
ERRO kubelet service is not running: ssh command error:
command : sudo systemctl -f is-active kubelet
err     : exit status 3
output  : inactive
```

Reply:
@cfergeau yes, I updated the PR after that, now it have another commit which should do it.

Reply:
It would be better to reorder the commits so that we don't break something in one of the early commits, and then fix it in a later one.